lib/rack/session/cookie.rb

```diff
@@ -51,11 +51,11 @@ class Cookie < Abstract::Persisted
       # Encode session cookies as Base64
       class Base64
         def encode(str)
-          [str].pack('m')
+          [str].pack('m0')
         end
 
         def decode(str)
```

Comment:
why not just delegate it to `Base64.strict_decode64(str)` which is in the stdlib?

https://github.com/ruby/ruby/blob/c5eb24349a4535948514fe765c3ddb0628d81004/lib/base64.rb#L73

Reply:
Hi,

That's what used to happen but was removed in https://github.com/rack/rack/commit/e6b65296a247e23742d85e47cf31582b5883918a for "1.9 support", whatever that means. I don't know if Rack still supports v1.9. The gemspec says not but I'm just following convention. Happy to change.

Regards,
iain

Reply:
by delagating to Base64 `str.unpack('m0').first` could be implemented as `str.unpack1("m0")` on modern rubies. If the original intention was 1.9 support maybe we could reconsider to restore the behaviour, the current version of rack supports at least ruby 2.2.

Reply:
Looking again, Rack's Base64 is a superclass to Marshal, JSON and ZipJSON. Any idea where ZipJSON is used? I don't see it being called anywhere in this lib.

Reply:
I also had a search through Github and beyond, I don't see anyone using those encoders.